app/views/search/results/partials/_protocol_text.html.erb

```diff
@@ -17,7 +17,7 @@
   </span>
   &nbsp;
 <% end %>
-<% if protocol.in_module? && can_view_protocol(protocol) || protocol.in_repository? && can_read_protocol_in_repository?(protocol) %>
+<% if can_read_protocol_in_module?(protocol) || can_read_protocol_in_repository?(protocol) %>
```

Comment:
where we declare this can_read_protocol_in_repository?

Reply:
In `/app/permissions/team.rb`.

Reply:
:+1: